app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -27,6 +27,11 @@
   <%= @application_choice.application_form.full_name %>
   <%= render(ProviderInterface::ApplicationStatusTagComponent.new(application_choice: @application_choice)) %>
 </h1>
+
+<%= render SubNavigationComponent.new(items: [{name: 'Application', url: provider_interface_application_choice_path(@application_choice), current: true},
+                                                        {name: 'Notes', url: '#'},
+                                                        {name: 'Timeline', url: '#'}]) %>
```

Comment:
If we remove this for now this PR can be merged before we have notes & timeline.

Reply:
Done